packages/kbn-check-mappings-update-cli/current_mappings.json

```
@@ -511,6 +511,22 @@
       },
       "FileKind": {
         "type": "keyword"
+      },
+      "hash": {
```

if you're not searching on `hash`, then rather leave it as unmapped:
```suggestion
      "hash": {
        "dynamic": false,
        "properties": {}
      }
```

`hash` will still be in `_source`.

---

Thanks for the feedback @TinaHeiligers. I will change the mapping to what you suggested.